skywalking-sniffer/skywalking-sdk-plugin/motan-plugin/src/main/java/com/a/eye/skywalking/plugin/motan/MotanConsumerInterceptor.java

```diff
@@ -13,37 +15,41 @@
 import com.weibo.api.motan.rpc.URL;
 
 /**
- * {@link MotanConsumerInvokeInterceptor} create span by fetch request url from
- * {@link EnhancedClassInstanceContext#context} and transport serialized context
- * data to provider side through {@link Request#setAttachment(String, String)}.
+ * Current trace segment will ref the trace segment from previous level if the serialized context data that fetch
```

Comment:
Did not understand. This is an interceptor, what is your meaning about `from previous level`.

I think this class is for tracing a motan service calling.